Dockerfile

```diff
@@ -22,3 +19,8 @@ RUN             apk add --virtual .build-deps \
                 apk add --virtual .run-deps \
                     ca-certificates && \
                 apk del .build-deps
+
+COPY            entrypoint.sh setup_creds.py /src/
+
+ENTRYPOINT      ["/src/entrypoint.sh"]
```

Comment:
It seems we need a permission for executing this file?
```
RUN             chmod +x /src/entrypoint.sh
```

Reply:
Yes, it was missing from the PR

Reply:
Updated, the file should have executing permissions. But I'm not sure where does GitHub show it.